### ISSUE TITLE
Provide Official ASK CLI Docker Image

### ISSUE THREAD

**Issue by Rabadash8820** [alexa/alexa-cookbook#98]
This probably isn't the correct repo to be posting this issue in; however, I couldn't find any repo for the ASK CLI in this organization, and there is no source code link on the `ask-cli` NPM page, so I was at a loss.  Anyway...

I was surprised to find that there is no official Docker image for development with the ASK CLI.  The "amazon" user on Docker Hub contains only the [these](https://hub.docker.com/u/amazon/) images for working specifically with ECS and Elastic Beanstalk, and there does not appear to be any official "alexa" user.  Upon further investigation, I found the [`aws-codebuild-docker-images`](https://github.com/aws/aws-codebuild-docker-images) repo in the "aws" GitHub organization, but unfortunately it does not have any sample Dockerfiles that install the ASK CLI either.  I created a [similar issue](https://github.com/aws/aws-cli/issues/3291) at the AWS CLI repo, because there is no official Amazon image for the AWS CLI either, so I imagine that whatever `aws-cli` image gets (hopefully) created could serve as the base for an `ask-cli` image.

**TL; DR**
I believe there should be an official `ask-cli` Docker image maintained by Amazon and pushed to Docker Hub/Store.  Ideally, it should support all the major modern Linux distros, including Alpine Linux.  One should be able to run the following command, and have everything just work:

`docker run -it alexa/ask-cli`

**Comment by franklin-lobb** [comment]
Hi @Rabadash8820, please post your request to [Alexa Skills Developer Voice](https://alexa.uservoice.com).  That site enables Alexa Skills Developers to request and vote on features you’d like to see in the developer toolset and services for Alexa.
